masterlist.yaml

```diff
@@ -5243,6 +5248,77 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/8210/' ]
     tag:
       - Relev
+  - name: 'EconomyOverhaulandSpeechcraftImprovements(Lite)?\.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/9542/' ]
+    after:
+      - 'Scarcity SE - Less Loot Mod.esp'
```

Comment:
Just realized that's loading fairly late, I'll have to make some changes to prevent a few sorting errors.

Reply:
Okay checked, and with the patch order doesn't matter.
Probably just outdated info on the mods description.
Just to be safe, remove that after and I can review scarcity's position.

![image](https://user-images.githubusercontent.com/1944639/59177839-363d9f80-8b55-11e9-9803-6288c9ac967a.png)